docs/commands/openfeature_init.md

```diff
@@ -15,9 +15,9 @@ openfeature init [flags]
 ### Options
 
 ```
-      --flag-source-url string   The URL of the flag source
-  -h, --help                     help for init
-      --override                 Override an existing configuration
+  -h, --help              help for init
+      --override          Override an existing configuration
+      --provider string   The URL of the flag provider
```

Comment:
I have a slight preference for `--provider-url` because `--provider` could be interrupted the provider name (e.g. flagd).